keras_nlp/models/gpt2/gpt2_presets.py

```diff
@@ -136,4 +136,34 @@
         "merges_url": "https://storage.googleapis.com/keras-nlp/models/gpt2_extra_large_en/v1/merges.txt",
         "merges_hash": "75a37753dd7a28a2c5df80c28bf06e4e",
     },
+    "gpt2_base_en_news": {
```

Comment:
I think the most similar checkpoint we have here is the `sst2` dataset for bert. So maybe we should stick to the more precise naming and call this, `gpt2_base_en_cnn_dailymail`?